src/Hyperledger.Aries/Configuration/DefaultProvisioningService.cs

```diff
@@ -44,7 +44,9 @@ public class DefaultProvisioningService : IProvisioningService
         }
 
         /// <inheritdoc />
-        public async Task AcceptTxnAuthorAgreementAsync(Wallet wallet, IndyTaa txnAuthorAgreement)
+        public Task AcceptTxnAuthorAgreementAsync(Wallet wallet, IndyTaa txnAuthorAgreement) => AcceptTxnAuthorAgreementAsync(wallet, txnAuthorAgreement, null);
+        /// <inheritdoc />
+        public async Task AcceptTxnAuthorAgreementAsync(Wallet wallet, IndyTaa txnAuthorAgreement, string acceptanceMechanism)
```

Comment:
@victorbalan you can make acceptanceMechanism an optional parameter and provide a default value of null